doc/web-api.md

```diff
@@ -219,6 +219,22 @@ a JSON object, containing:
     * error_line: line of the given XML where the error was found (None if no error was found or if it was not a validation error)
 
 
+#### POST /v1/on_same_fs
+
+Check if a given filename exists and if first 32 bytes of its content  has the same checksum passed as argument of POST.
```

Comment:
`if first 32 bytes of its content  has` -> `if the first 32 bytes of its content have`